src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -78,16 +78,16 @@ public class GetAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         /// <summary>
         /// State of protection of the item to be returned.
         /// </summary>
-        [Parameter(Mandatory = false, Position = 4, HelpMessage = ParamHelpMsgs.Item.Status)]
```

Comment:
You cannot remove a position from a parameter as this is a breaking change.  Please add the position back.  If you would like to deprecate this position, feel free to do so and it will be removed at the breaking change release in November: https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-attribute-help.md